harvard-lts/CURIOSity
# Minor fixes

**Some minor fixes**
* * *


# What does this Pull Request do?
Removes the old json libraries since we upgraded, bumps another vulnerable library, makes a DockerfileLocal so we can have vim on local work

# How should this be tested?

Spin this up locally and make sure everything works and also you should have vim in your local now. The image is also on dev (not with vim though as that doesn't use DokcerfileLocal)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests?

# Interested parties
Tag (@ mention) interested parties
